great-idea/index.html

```diff
@@ -14,10 +14,89 @@
   <![endif]-->
 </head>
 
-  <body>
+<body>
 
-    <!-- Copy and paste your HTML from the first UI project here -->
-      
-    
-  </body>
+  <div class="navigation">
+    <nav>
+        <label>Services</label>  
```

Comment:
When dealing with navbar menus, it is best to use the `<a>` tag to make the menus clickable. 😄